readme.md

```diff
@@ -167,7 +167,7 @@
 - [Fortran](https://github.com/rabbiabram/awesome-fortran#readme)
 - [PHP](https://github.com/ziadoz/awesome-php#readme)
 	- [Composer](https://github.com/jakoch/awesome-composer#readme) - Package manager.
-- [Delphi](https://github.com/Fr0sT-Brutal/awesome-delphi#readme)
+- [Pascal](https://github.com/Fr0sT-Brutal/awesome-pascal#readme) - Delphi/FreePascal/(any)Pascal
```

Comment:
Needs a better description.

Reply:
What else description do you want? Language lists in common do not have them at all and I provided one. Write your own if it makes you satisfied

Reply:
> What else description do you want?

Read the contribution guidelines:

> The entry in the Awesome list should:
> * Include a short description about the project/theme of the list. **It should not describe the list itself.**

Reply:
It's still not getting clear. It's a list for a programming language. Theme of the list is a programming language. The language is mentioned in the list name. Moreover **none** of other programming language lists contain a description except for several exotic and little-known ones. Why do you demand special requirements personally for my list??
And again. What kind of description do you demand? Will `Pascal is an imperative and procedural programming language, which Niklaus Wirth designed in 1968–69 and published in 1970, as a small, efficient language intended to encourage good programming practices using structured programming and data structuring. It is named in honor of the French mathematician, philosopher and physicist Blaise Pascal.` satisfy you?

Reply:
> Why do you demand special requirements personally for my list??

It's not specially for your list. It's for all new contributions.

> satisfy you?

No need to be snarky.